nerdtribe/mr-binge
# Media Card and Basic List

## Explanation of work done and why:


Adds generic media card, media component, views and stylistic changes for MVP.
Movies: ![image](https://user-images.githubusercontent.com/8315447/66593537-3383a300-eb65-11e9-96f5-24042eee412d.png)
TV: ![image](https://user-images.githubusercontent.com/8315447/66593554-3b434780-eb65-11e9-8519-8ad9ca8ff6d2.png)


## How to (manually) test:

1. Fetch `git fetch && git checkout -b media-card-list origin/media-card-list`
2. Serve `yarn electron:serve`
3. Observe the cards are generated for different views (movie & tv).

## Checklist:
Closes #8, #7
